### ISSUE TITLE
[Bug] Resume Active Download is not working for private group chatID

### ISSUE THREAD

**Issue by cutesniper22** [JMax45/telegram-bulk-downloader#1]
I tried bulk download all items from private group by adding "-100 + chatID"
e.g : if let say private group chat id = 2121461301, then i added as -1002121461301 (just a dummy chatID)
It is working well on "start new download" after assigned private group chatID.

But the issue/error occurred when resume active download for this existing chatID.
I found that chatID still remained as 2121461301 (without 100?) under "AppData\Roaming\TelegramBulkDownloader\state" file , therefore suspect it cannot find the actual chatID.

Appreciate your help on solving this issue.

**Comment by JMax45** [comment]
Hi, thank you for your detailed report.

I've managed to replicate the issue you're experiencing, and you're right—the missing "-" when resuming the download was indeed causing problems. I've addressed this in the latest version (1.1.2).

This version now ensures that the original chatID string you provide (including the "-") remains unchanged. Once you update to 1.1.2 you'll need to start a new download for that specific chatID, as the previous saved state will be incompatible with the new fix.

Let me know if this fixes the problem, and thanks again for your help in improving the tool!
